salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
I agree the documentation can be more concise, but I think it is important to be explicit about how the `pillar_merge_lists` config option modifies the behavior of `merge` in `pillar.get()`. That was unexpected behavior to me. I expect `pillar_merge_lists` to work against dictionaries within pillar, yes, but not within dictionaries in my states.